test/errors.js

```diff
@@ -49,7 +50,9 @@ var HTTP_CODES = [
     { name: 'UnprocessableEntityError', code: 422 },
     { name: 'LockedError', code: 423 },
     { name: 'FailedDependencyError', code: 424 },
-    { name: 'UnorderedCollectionError', code: 425 },
+    { name: http.STATUS_CODES[425] === 'Too Early'
+      ? 'TooEarlyError'
+      : 'UnorderedCollectionError', code: 425 },
```

Comment:
is backward compatibility really necessary here?

Reply:
The status code name comes from Node.js core, so we have no control over it.